src/test/java/duke/TodoTest.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TodoTest {
+    @Test
+    public void toStringTest() {
+        Todo temp1 = new Todo("", false);
```

Comment:
Do you think there can be a better test variable name to use instead of _temp1_?